### ISSUE TITLE
Error while trying to run npm run test:e2e 

### ISSUE THREAD

**Issue by matiasbavera** [osrf/romi-dashboard#138]
Error while trying to run on local env
```
npm run test:e2e 
. e2e/env.sh && scripts/dockert docker-compose -f docker/docker-compose.yml up e2e
docker-compose -f docker/docker-compose.yml up e2e
```

```
e2e_1        | npm WARN prepare removing existing node_modules/ before installation
e2e_1        | npm ERR! Error while executing:
e2e_1        | npm ERR! /usr/bin/git ls-remote -h -t git://github.com/RobotWebTools/rclnodejs.git
e2e_1        | npm ERR! 
e2e_1        | npm ERR! fatal: failed to stat '/root/romi-dashboard/packages/dashboard': Permission denied
e2e_1        | npm ERR! 
e2e_1        | npm ERR! exited with error code: 128
```

`docker-compose -f docker/docker-compose.yml up dev
`it's working correctly

I think it has something to do with the permissions when executing npm ci.

**Comment by matiasbavera** [comment]
It also breaks the local installation.

**Comment by matiasbavera** [comment]
If I run npm ci in advance on dashboard and dashboard/e2e I have a different error. Should we give permission to the container script to modify a file outside the docker?

**Comment by koonpeng** [comment]
It's a known issue with npm, it's not using the correct EUID. Make sure your permissions and owners is not "tainted".

**Comment by koonpeng** [comment]
can be closed when #151 is merged, if that works out, we will no longer be using docker to launch e2e tests.

**Comment by ChawinTan** [comment]
Merged
